### ISSUE TITLE
Mob souls put into chests break the whole GUI of the game

### ISSUE THREAD

**Issue by Grynder01** [brandon3055/Draconic-Evolution#1172]
## Base information
If i put mob souls into any chest it breaks the GUI of the entire game, hotbar item are sometime invisible though they are still there and the same goes for items in chests and when you open the player inventory items are all over the screen and the inventory gui is invisible
also make the game menu invisible although it is still there and usable if you can guess where the buttons are 
* Minecraft version:1.10.2
* Mod version:2.1.6.263
* Minecraft Forge version:12.18.3.2477
* Mod Pack: (if applicable)sky factory 3 v3.0.15

**Comment by covers1624** [comment]
Please provide some screenshots, Basically all you are telling us here, is 'some random mob soul breaks rendering in inventory, fix it'. We have absolutely no idea _what_ mob soul is causing it, nor do we know the actual affects you are describing.

**Comment by Grynder01** [comment]
Ok I will see about trying to get some screen shots
I'm sorry if it seemed like I was just like FIX IT I was more hoping it was something you had already encountered. 
The last time it happened it was a spider soul
but I cant be sure it is limited to just that one
Its not like I put a soul in a chest and BANG things go wonky sometimes its ok for minutes or even hours
it seems to be related to opening the chests with the souls already in there
At first it happened in an ender chest and I thought it was specific to the souls and ender chests
so I loaded the backup and changed the ender chest for a diamond chest but it happened again
So that made me think it was possibly related to being added to the chest by an advanced item collector
so again I reloaded a backup and took away the item collector and used individual iron chests for each soul type (just as an extra layer of precaution) and I was going through the chests trying to find the zombie souls so I had opened 2 or 3 chests when I came to the spider chest as soon as I opened it the wonky was triggered
So I loaded the backup again and just got past the situation by keeping all the souls in my player inventory until I was ready to place the spawners ( I have 8 total running right now)
So to sum it up I'm past it now but I would still like to help isolate the problem so I'm going to try to make a dedicated backup and get it to happen again so I can show screens

**Comment by Grynder01** [comment]
![2018-09-19_19 34 26](https://user-images.githubusercontent.com/43357639/45789306-a19cb580-bc44-11e8-8afd-9c2139a2a039.png)
![2018-09-19_19 34 27](https://user-images.githubusercontent.com/43357639/45789307-a3ff0f80-bc44-11e8-9008-cc8c228c5d4d.png)
![2018-09-19_19 34 28](https://user-images.githubusercontent.com/43357639/45789311-a5c8d300-bc44-11e8-84c2-c7c3353992d9.png)
![2018-09-19_19 34 30](https://user-images.githubusercontent.com/43357639/45789313-a7929680-bc44-11e8-9ddf-86085cea52d4.png)
![2018-09-19_19 34 33](https://user-images.githubusercontent.com/43357639/45789319-ae210e00-bc44-11e8-8e7c-27e293442a3a.png)
![2018-09-19_19 41 49](https://user-images.githubusercontent.com/43357639/45789334-b2e5c200-bc44-11e8-80db-b6360f82561c.png)
![2018-09-19_19 41 52](https://user-images.githubusercontent.com/43357639/45789337-b6794900-bc44-11e8-9086-88b124c73d8b.png)
![2018-09-19_19 42 04](https://user-images.githubusercontent.com/43357639/45789340-b8430c80-bc44-11e8-80b9-03e1fb9ef505.png)
![2018-09-19_19 42 08](https://user-images.githubusercontent.com/43357639/45789343-baa56680-bc44-11e8-8c04-18caf6dcf962.png)
![2018-09-19_19 42 10](https://user-images.githubusercontent.com/43357639/45789345-bc6f2a00-bc44-11e8-9467-34f5664d68e7.png)

**Comment by Grynder01** [comment]
Also i seem to remember from an earlier version all the souls started as pig souls or all looked like pig souls or something like that...I don't recall exactly. So I'm now wondering if it has something to do with getting a soul from some weird added spawn that is not a vanilla mob and it breaking things

**Comment by Grynder01** [comment]
Ok so sorry for all the double posting BUUUUT
On my last thought there I decided to go back and do further testing and can confirm that its not the souls in chests that matter 
If you can find the Invisible pig soul in your inventory and drop it the gui fixes itself
And I went on and gathered many more souls and put them in chests...NO PROBLEM
The thing is I was picking up the soul and then the first time I opened my inventory again it was to look in one of the chests so I thought it had something to do with the chests but now I know it has nothing to do with storing souls in chests
If you just keep going eventually you will get that random pig soul again
and everything gets all messed up again 
So I decided to go put that soul in a spawner to see what i got and low and behold these are the screens
![2018-09-19_20 33 23](https://user-images.githubusercontent.com/43357639/45790800-2d661000-bc4c-11e8-9cca-07a08ebaa33a.png)
![2018-09-19_20 33 27](https://user-images.githubusercontent.com/43357639/45790805-33f48780-bc4c-11e8-97ca-5f98e66aaeb7.png)
![2018-09-19_20 33 34](https://user-images.githubusercontent.com/43357639/45790806-35be4b00-bc4c-11e8-92d4-09437860730f.png)
![2018-09-19_20 33 38](https://user-images.githubusercontent.com/43357639/45790808-37880e80-bc4c-11e8-83e5-87c5d854d07d.png)

**Comment by brandon3055** [comment]
Oh if i had a dollar for every issue i have had related to Headcrumbs.... I dont suppose you checked your console? It should be spammed with errors from the mob soul renderer.

**Comment by Grynder01** [comment]
Awesome
im done for tonight but ill try to find a download and test tomorrow
